darts-api-stub/annotations/annotation.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const { localArray } = require('../localArray');
+const path = require('path');
+
+const router = express.Router();
+
+const resBody104 = {
+  type: 'CASE_104',
+  title: 'The requested document cannot be found',
+  status: 404,
+};
+
+const getAnnotationDocumentByAnnotationAndDocumentId = (annotationId, annotationDocumentId) => {
+  const annotations = localArray('annotations');
+  const annotation = annotations.value.find((annotation) => annotation.annotation_id === parseInt(annotationId));
+  if (!annotation) return null;
+  return annotation.annotation_documents.find(
```

Comment:
shouldn't this just be `return annotation` local variable?

Reply:
This line is looking for the document ID in the same annotation it found in the first find. If either the annotation or document are not found it will return `undefined` which will satisfy the check on line 28 and return a 404

Reply:
Ah, ok that makes more sense, thank you